app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/evaluation/scenario/Scenarios.tsx

```
@@ -0,0 +1,1439 @@
+function Scenario(props) {
+  const {
+    id,
+    data,
+    loadScenario,
+    mode,
+    loading,
+    differenceMode,
+    selectedSubsets,
+    loadView,
+  } = props;
+  const { key, compareKey } = data?.view;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)